generator/template/sql_builder_template.go

```diff
@@ -145,11 +146,42 @@ func DefaultTableSQLBuilderColumn(columnMetaData metadata.Column) TableSQLBuilde
 // getSqlBuilderColumnType returns type of jet sql builder column
 func getSqlBuilderColumnType(columnMetaData metadata.Column) string {
 	if columnMetaData.DataType.Kind != metadata.BaseType &&
-		columnMetaData.DataType.Kind != metadata.RangeType {
+		columnMetaData.DataType.Kind != metadata.RangeType &&
+		columnMetaData.DataType.Kind != metadata.ArrayType {
 		return "String"
 	}
 
-	switch strings.ToLower(columnMetaData.DataType.Name) {
+	typeName := columnMetaData.DataType.Name
+	columnName := columnMetaData.Name
+
+	if columnMetaData.DataType.Kind == metadata.ArrayType {
```

Comment:
Why not add a case switch for each of the array types, as it is already done for other types?

Reply:
I added a new `sqlArrayToColumnType` to convert a type to an array type. The function switches on the type name without the `[]` suffix because you would have to add infinite brackets for multidimensional arrays.

Reply:
When I think about it again it makes sense to separate arrays from `sqlToColumnType`. Since every postgres type can be element of array(`[]bool`, `[]point`, `[]timestamp`, etc...), sql builder array type can be constructed with just  `sqlToColumnType`:

```golang
columnType = sqlToColumnType(strings.TrimSuffix(typeName, "[]")) + "Array"
```

Reply:
This can lead to incorrectly generated code, because this PR does not support DateArray for example. What's your opinion on this?

Reply:
It can be supported with one line `type DateArray Array[DateExpression]`. For model type we can fallback to string if there is no `pq` date array type.